pkg/services/sqlstore/migrator/conditions.go

```diff
@@ -46,3 +46,28 @@ type IfColumnNotExistsCondition struct {
 func (c *IfColumnNotExistsCondition) SQL(dialect Dialect) (string, []interface{}) {
 	return dialect.ColumnCheckSQL(c.TableName, c.ColumnName)
 }
+
+type IfColumnExistsCondition struct {
+	ExistsMigrationCondition
+	TableName  string
+	ColumnName string
+}
+
+func (c *IfColumnExistsCondition) SQL(dialect Dialect) (string, []interface{}) {
+	return dialect.ColumnCheckSQL(c.TableName, c.ColumnName)
```

Comment:
I am surprised this is the same as the IfColumnNotExistsCondition SQL a few lines above 🤔